check/check.go

```diff
@@ -210,7 +210,23 @@ func textToCommand(s string) (cmds []*exec.Cmd) {
 
 	cp := strings.Split(s, "|")
 
+	var newCP []string
+	joinNextPart := false
+
+	// Separate between pipeline and or, differentiate by | and \\|
```

Comment:
```suggestion
	// Generally we split into commands separated by |
	// However, if the | was preceded by \\ then we need to undo that split, and replace the | (without the backslashes) 
```

I think this comment would be a bit clearer